src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1016,6 +1016,12 @@ private static bool IsPackageReferenceAutoReferenced(PackageReference package)
                         preferredVersions.Remove(packageId);
                     }
                 }
+                else
+                {
+                    // we return an empty list here because this is trying to update a specific package in a project here which is not even installed in the project.
+                    // Bug: https://github.com/NuGet/Home/issues/737
+                    return new List<NuGetProjectAction>();
```

Comment:
does it change any behavior for `update-package -id packageA -projectName projectA` when packageA is not installed in that project?